<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grabar Prueba</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/btn1.png</normaloff>../resources/btn1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCamOjo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCamEscena">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cámara 1. OJO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblText_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cámara 2. ESCENA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoIpn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/logoipn.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoEscom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/logoescom.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGrabar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>570</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/btn5.png</normaloff>../resources/btn5.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/btn6.png</normaloff>../resources/btn6.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnalizar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Analizar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/btn2.png</normaloff>../resources/btn2.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblhrs">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblmin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblseg">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblhrs_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:hrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblmin_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:min</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblseg_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:s</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progBar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>590</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcess">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>560</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
